etc/config/compiler-explorer.defaults.properties

```diff
@@ -11,6 +11,7 @@ supportsExecute=true
 optionsBlacklistRe=^(-W[alp],)?((--?(wrapper|fplugin.*|specs|load|plugin|include)|(@.*)|-I|-i)(=.*)?|--)$
 allowedShortUrlHostRe=^([-a-z.]+\.)?(xania|godbolt)\.org$
 googleShortLinkRewrite=^https?://goo.gl/(.*)$|https://godbolt.org/g/$1
+urlShortenService=none
```

Comment:
I think the default one should be `google` instead of none, if only because that's what the rest of the values in this file are doing. They are set so (most of them) work for us out of the box, and then 3rd party users just need to create their overriding files

Reply:
Yes, I was unsure what to use as default value.

The reason i chose `none` was because I thought that was closer to the previous behavior of the default config. As the default config didn't provide an `googleApiKey` it would end up with url shortening disabled.

But I can change it to `google` if that means less hassle.

Reply:
I agree with that default should be none, but should be google in compiler-explorer.amazon.properties

Reply:
I now see how none could be a sensible choice, yes. Note that we would also need to add the google shortner to `compiler-explorer.beta.properties`

Reply:
Updated those files in latest commit. (however, noted that the `beta` file don't have any `googleApiKey` which the `amazon` file has)